### PR TITLE
div. Features

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: grails-app/services/de/laser/SubscriptionService.groovy [hunk 1]
```diff
@@ -1149,6 +1149,8 @@ class SubscriptionService {
     boolean deleteEntitlementbyID(sub, id) {
         IssueEntitlement ie = IssueEntitlement.findWhere(id: Long.parseLong(id), subscription: sub)
         if(ie == null) {
+            println(id)
```
Comment (croqueGrec09) [inline grails-app/services/de/laser/SubscriptionService.groovy:1152]: needed?

**Review by croqueGrec09 — COMMENTED** [review]
one remark

**Comment by Moetez16** [comment]
Done!

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**